great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -98,35 +109,40 @@ def __init__(
     ) -> None:
         """
         Args:
-            name: the name of this parameter -- this is user-specified parameter name (from configuration);
-            it is not the fully-qualified parameter name; a fully-qualified parameter name must start with "$parameter."
-            and may contain one or more subsequent parts (e.g., "$parameter.<my_param_from_config>.<metric_name>").
+            name: the name of this parameter -- this is user-specified parameter name (from configuration); it is not
+                the fully-qualified parameter name; a fully-qualified parameter name must start with "$parameter."
```

Comment:
@NathanFarmer what is being changed here?  Just the indentation?  Do we need it?  Thanks.